src/providers/local.py

```
@@ -1792,6 +1870,74 @@ async def stop_session(self):
+            for task in tasks
+            if task is not None and task is not current and not task.done()
+        ]
+        for task in pending_tasks:
+            task.cancel()
+
+        ws = self.websocket
+        self.websocket = None
+        if ws is not None:
+            try:
+                await ws.close()
+            except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)